.gitignore

```diff
@@ -57,5 +35,7 @@ typings/
 # dotenv environment variables file
 .env
 
-# next.js build output
-.next
+# compiled lib
+build
+
+BUILD_SHA
```

Comment:
There's no need to join the lock files for a plugin package ^^
```suggestion
BUILD_SHA

# lock files
yarn.lock
package-lock.json

```

Reply:
It's true that they don't apply to the end consumers of the package, but I kind of like having them committed so that all contributors are working with the same versions in development and you don't get hiesenbugs depending on a developer's environment or when they cloned the package.